libs/wingsdk/src/shared-aws/api.inflight.ts

```diff
@@ -0,0 +1,6 @@
+import { IApiClient } from "../cloud";
+
+export class ApiClient implements IApiClient {
+  // TODO: does `url` belong inside the inflight class?
+  public constructor(public readonly url: string) {}
+}
```

Comment:
In order to get `api.url` accesses to work on AWS, I needed to copy the field from preflight into inflight. Something smells off to me about this, but I think it's probably reasonable for now since this is also a kind of duplication the compiler does when jsify-ing user-defined classes I think

Reply:
Wouldn't it make sense to pass it though and environment variable like we pass other preflight data to inflight clients?

I feel I might be missing something...